### DISCUSSION TITLE
Consider to disable / pause the dependbot to avoid flush the comits history with no useful info

### DISCUSSION THREAD

**Discussion by royalgarter** [mongo-express/mongo-express#1081] [Polls]
As mention

**Comment by rtritto** [comment]
A good solution is to enable dependabot on a new branch and then merge it using a single commit (squash).

Waiting @shakaran @BlackthornYugen

**Reply by rtritto** [reply]
Maybe [Renovate](https://github.com/renovatebot/renovate) can help (a check is needed) to use a single commit.

**Comment by rtritto** [comment]
Dependabot is improved with dependencies group.
